google/cloud/spanner/client_options.h

```diff
@@ -35,6 +36,12 @@ class ClientOptions {
   ClientOptions(ClientOptions&&) = default;
   ClientOptions& operator=(ClientOptions&&) = default;
 
+  /**
+   * Convert the `ClientOptions` to the new, recommended way to represent
+   * options of all varieties, `google::cloud::Options`.
+   */
+  explicit operator Options() const;
```

Comment:
in pubsub a conversion method like this was hidden away in `internal` with the logic being that `ClientOptions` was to be deprecated, so no `public` methods should be added to it.

Reply:
I was thinking that a customer might want to call this as part of their own migration, but I'm just guessing.  In any case, with `ClientOptions` on the road to removal I'm not too concerned about adding to the API.